scripts/editor/icons/icons.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+// import React from 'react';
```

Comment:
Why is this commented out? UI icons are JSX SVGs

Reply:
Was testing out something when my tests were failing, forgot to return this, good catch.